plugins/lookup/hashi_vault.py

```diff
@@ -511,6 +543,18 @@ def auth_methods(self):
         if hasattr(self, auth_validator) and callable(getattr(self, auth_validator)):
             getattr(self, auth_validator)(auth_method)
 
+    def action_types(self):
+        # enforce and set the list of available hvac action types
+        # TODO: can this be read from the choices: field in documentation?
+        avail_action_types = ['READ', 'LIST']
+        self.set_option('avail_action_types', avail_action_types)
```

Comment:
I don't think the available action types should be set as an option, since It's not read anywhere.

Reply:
Thanks, removed in [this commit](https://github.com/ansible-collections/community.general/pull/664/commits/c9240e9d70758ff297af986d54a0e42a664e0844)